alacritty/src/input.rs

```diff
@@ -366,23 +366,66 @@ impl<'a, T: EventListener, A: ActionContext<T>> Processor<'a, T, A> {
         }
     }
 
-    pub fn normal_mouse_report(&mut self, button: u8) {
+    pub fn utf8_mouse_report(&mut self, button: u8) {
         let (line, column) = (self.ctx.mouse().line, self.ctx.mouse().column);
 
-        if line < Line(223) && column < Column(223) {
-            let msg = vec![
+        if line < Line(2015) && column < Column(2015) {
+            let mouse_pos_encode = |pos: usize| -> (u8, u8) {
```

Comment:
If we return a slice here, we can directly append all elements to `msg` instead of having to use this tuple.